stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -122,6 +278,8 @@ extension DistributedActor /*: implicitly Decodable */ where Self.ID: Decodable
 
 @available(SwiftStdlib 5.7, *)
 extension DistributedActor /*: implicitly Encodable */ where Self.ID: Encodable {
+
+  /// Encodes the ``actor.id`` as a single value into the passed `encoder`.
```

Comment:
```suggestion
  /// Encodes the `actor.id` as a single value into the passed `encoder`.
```

? Or \`\`actor.id\`\` works?

Reply:
good catch, that would not work - thanks!